unittests/resources/xml/test_trim.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Maybe we should name the file `test_fm_with_whitespaces.xml` with just trim it's not really clear why this file is special.